src/main/kotlin/no/nav/tilleggsstonader/sak/interntVedtak/InterntVedtak.kt

```diff
@@ -13,12 +16,17 @@ import java.time.LocalDate
 import java.time.LocalDateTime
 import java.util.UUID
 
+/**
+ * Vurder å ta med fakta
+ */
```

Comment:
Lettere å søke opp om vi faktisk skal fikse det noen gang
```suggestion
/**
 * Todo: Vurder å ta med fakta
 */
```